WayMovies/Controller/BrowseViewController.swift

```diff
@@ -0,0 +1,399 @@
+            searchBar.leftAnchor.constraint(equalTo: view.leftAnchor),
+            searchBar.rightAnchor.constraint(equalTo: view.rightAnchor),
+            searchBar.heightAnchor.constraint(equalToConstant: searchBarHeight)
+        ])
+    }
+    
+    /**
+      Sets up constraints for the suggestion table
+    */
+    private func suggestionTableConstraints() {
+        NSLayoutConstraint.activate([
+            suggestionTable.topAnchor.constraint(equalTo: searchBar.bottomAnchor),
+            suggestionTable.leftAnchor.constraint(equalTo: view.leftAnchor),
+            suggestionTable.rightAnchor.constraint(equalTo: view.rightAnchor),
+            suggestionTable.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+    }
+    
+    /**
+     Creates and constrains the collections in the collection list
+     */
+    private func createCollectionsAndConstraints() {
+        for (index, collection) in collectionList.enumerated() {
+            scrollView.addSubview(collection)
+            if index == 0 {
+                NSLayoutConstraint.activate([
+                    collection.topAnchor.constraint(equalTo: scrollView.topAnchor),
+                    collection.leftAnchor.constraint(equalTo: view.leftAnchor),
+                    collection.rightAnchor.constraint(equalTo: view.rightAnchor),
+                    collection.heightAnchor.constraint(equalToConstant: collectionHeight)
+                ])
+            } else {
+                NSLayoutConstraint.activate([
+                    collection.topAnchor.constraint(equalTo: collectionList[index-1].bottomAnchor),
+                    collection.leftAnchor.constraint(equalTo: view.leftAnchor),
+                    collection.rightAnchor.constraint(equalTo: view.rightAnchor),
+                    collection.heightAnchor.constraint(equalToConstant: collectionHeight)
+                ])
+            }
+        }
+    }
+    
+    /**
+     Navigates to the favorites page of the item associated with the cell
+     */
+    @objc func viewFavorites(sender: UIBarButtonItem) {
+        navigationController?.pushViewController(FavoritesViewController(), animated: true)
+    }
+    
+//    @objc func refresh() {
+//        for collection in collectionList {
+//            collection.TVCollection.reloadData()
+//        }
+//        refreshControl.endRefreshing()
+//    }
+    
+}
+
+extension BrowseViewController: NavigationDelegate {
+    /**
+     Called when the cell is tapped to navigate to the details page of the item
+     */
+    func cellTapped(_ item: TVItem) {
+        navigationController?.pushViewController(DetailViewController(item: item), animated: true)
+    }
+}
+
+extension BrowseViewController: UITableViewDelegate, UITableViewDataSource {
+    /**
+     Returns the number of suggested categorized search options
+     */
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 3
```

Comment:
If this table is always 3 rows and no cell reuse is needed, strongly consider switching to StackView